fiona/ogrext.pyx

```diff
@@ -1663,65 +1659,75 @@ def remove_virtual_file(vsi_filename):
     return VSIUnlink(vsi_cfilename)
 
 
-cdef class MemoryFileBase(object):
+cdef class MemoryFileBase:
     """Base for a BytesIO-like class backed by an in-memory file."""
 
```

Comment:
Would it be an option to put `cdef VSILFILE * _vsif`here before the init() instead of the new MemoryFileBase C class in ogrext.pxd?

Reply:
That's a good suggestion. I'll try it.

Reply:
🎉